ibm_catalog.json

```diff
@@ -1,201 +1,216 @@
+            "profiles": [
+              {
+                "profile_name": "IBM Cloud Framework for Financial Services",
+                "profile_version": "1.7.0"
+              }
+            ]
+          },
+          "iam_permissions": [
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "service_name": "all-account-management-services"
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Writer",
+                "crn:v1:bluemix:public:iam::::role:Manager"
+              ],
+              "service_name": "container-registry"
+            }
+          ],
+          "architecture": {
             "features": [
-                {
-                    "title": "Creates or Uses Existing IBM Container Registry Namespace",
-                    "description": "Enables the creation of a new IBM Container Registry namespace or the use of an existing one, allowing users to define isolated environments for managing container images, with the ability to set a retention policy."
+              {
+                "title": "Creates or uses an existing IBM Container Registry namespace, configures pull traffic and storage quotas, and supports upgrading the registry plan to Standard.",
+                "description": "This architecture creates or utilizes an existing IBM Container Registry namespace, provides the ability to configure pull traffic limits and storage quotas in megabytes, and allows for upgrading the registry plan to Standard. It ensures efficient management of container image access by regulating data pull volume from the registry and setting storage capacity limits for container images within each registry."
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "caption": "Creates IBM Container Registry namespace.",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-container-registry/main/reference-architecture/deployable-architecture-icr.svg",
+                  "type": "image/svg+xml"
                 },
+                "description": "This architecture creates or utilizes an existing IBM Container Registry namespace, provides the ability to configure pull traffic limits and storage quotas in megabytes, and allows for upgrading the registry plan to Standard. It ensures efficient management of container image access by regulating data pull volume from the registry and setting storage capacity limits for container images within each registry."
+              }
+            ]
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "prefix",
+              "required": true
+            },
+            {
+              "key": "existing_resource_group_name",
```

Comment:
This should have a display name `resource_group` and there should be a single dependency on account settings copy and paste this one https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/main/ibm_catalog.json#L272-L304
just update the output to use `workload_resource_group_name`